src/meta-srv/src/procedure/region_failover/activate_region.rs

```diff
@@ -43,6 +45,7 @@ pub(super) struct ActivateRegion {
     remark_inactive_region: bool,
     region_storage_path: Option<String>,
     region_options: Option<HashMap<String, String>>,
+    wal_options_map: Option<HashMap<RegionNumber, EncodedWalOptions>>,
```

Comment:
Empty map implies `None` already:

```suggestion
    wal_options_map: HashMap<RegionNumber, EncodedWalOptions>,
```

Reply:
But `region_options` is of type `Option<HashMap>`. No need to follow the coding style?

Reply:
Well, we could change `region_options`, too.

Reply:
> Well, we could change `region_options`, too.
> But `region_options` is of type `Option<HashMap>`. No need to follow the coding style?

Here, the `None` stands for uninitialized. Could you add some comments?

Reply:
Added a comment: ` // An "None" option stands for unintialized. `